x/paramHub/genesis.go

```diff
@@ -71,6 +71,10 @@ const (
 	MiniIssueFee   = 3e8
 	MiniSetUriFee  = 37500
 	MiniListingFee = 8e8
+
+	// Cross stake fee
+	CrossDistributeRewardRelayFee      = 2e6 // 0.02 BNB
```

Comment:
0.006 BNB is good enough.

Reply:
You can refer to this API: https://dex-asiapacific.binance.org/api/v1/fees

Reply:
OK